llvm/test/CodeGen/AIE/aie2/GlobalISel/legalize-dyn-stackalloc.ll

```diff
@@ -92,23 +94,24 @@ define void @test_loop_dyn_alloca(i32 noundef %n) {
 ; CHECK-NEXT:    nop // Delay Slot 4
 ; CHECK-NEXT:    nop // Delay Slot 3
 ; CHECK-NEXT:    nop // Delay Slot 2
-; CHECK-NEXT:    nop // Delay Slot 1
+; CHECK-NEXT:    mov sp, p6 // Delay Slot 1
```

Comment:
interesting. That was pretty wrong, was it not?

Reply:
Looks like it yes! Then again, I'm not sure I understand this test fully.